code/modules/mob/mob_defines.dm

```diff
@@ -54,6 +54,8 @@
 	var/eye_damage = 0//Living, potentially Carbon
 	var/lastpuke = 0
 
+	var/circulatory = 0 // Some simple mobs will receive damage from disabler bolts
```

Comment:
Literally I'm 12 and what is this.

I've never actually done anything more than just mess around with the code so if you could let me know what the problem is and how better to accomplish this, that'd be great?